packages/utils-tests-helpers/src/hint-runner.ts

```diff
@@ -102,6 +102,7 @@ const validateResults = (t: ExecutionContext<HintRunnerContext>, results: Proble
         const found = reportsCopy.some((report, i) => {
             index = i;
 
+
```

Comment:
remove extra empty line.

Reply:
This was done in purpose to trigger a full build. I'm doing tests